tox.ini

```diff
@@ -50,6 +50,15 @@ usedevelop = true
 deps =
     -r{toxinidir}/requirements.txt
 
+# This is used for Windows, since binary name is different
+[testenv:mkvenv-win]
+basepython = python.exe
```

Comment:
You can install multiple versions of Python on Windows in parallel, e.g. having both `C:\Python2.7\python.exe` and `C:\Python3.5\python.exe`, so it's possible to have Python 2 in your path and still run qutebrowser with Python 3 (as tox can copy the Python 3 into the venv, regardless if it's in the path or not).

But fair point, it's a mess anyway and we need to have a compromise, so having it work for most users who install just Python 3 is probably okay. The rest can edit `basepython` to point to their Python 3 version :wink: